synthtool/gcp/templates/python_library/noxfile.py.j2

```diff
@@ -229,7 +229,7 @@ def docfx(session):
     """Build the docfx yaml files for this library."""
 
     session.install("-e", ".")
-    session.install("sphinx==4.0.1", "alabaster", "recommonmark", "gcp-sphinx-docfx-yaml")
+    session.install("sphinx==4.0.1", "alabaster", "recommonmark", "gcp-sphinx-docfx-yaml", "sphinx-copybutton")
```

Comment:
The `docfx` session generates YAML files, not HTML, so this change (and the next one) are unnecessary.

Reply:
As Tres mentioned, `docfx` generates YAML and thus we need additional support for this extra plugin in `https://github.com/googleapis/sphinx-docfx-yaml`. Rest assured, the YAML being generated using this plugin will have click-to-copy buttons enabled on code snippets.

As such, please revert this for `docfx` session.

Reply:
Thanks for the helpful comments. I've reverted the changes for the`docfx` session.